libts/state.go

```diff
@@ -17,9 +19,8 @@ type State struct {
 	Prefs *ipn.Prefs
 
 	// Current Tailscale backend state.
-	//  "NoState", "NeedsLogin", "NeedsMachineAuth", "Stopped",
-	//  "Starting", "Running".
```

Comment:
these comments will go out of date real quick with no automated tooling (there are non AFAIK). so better remove it and lookup enums based on type in VS Code (single source of truth is enum/type definition)